va/va.h

```diff
@@ -551,6 +592,23 @@ typedef struct _VAConfigAttrib {
 #define VA_DEC_SLICE_MODE_NORMAL       0x00000001
 /** \brief Driver supports base mode for slice decoding */
 #define VA_DEC_SLICE_MODE_BASE         0x00000002
+
+/** @name Attribute values for VAConfigAttribDecJPEG */
+/**@{*/
+typedef union _VAConfigAttribValDecJPEG {
+    /** \brief Set to (1 << VA_ROTATION_xxx) for supported rotation angles. */
+    unsigned int rotation;
+    /** \brief Reserved for future use. */
+    unsigned int reserved[3];
```

Comment:
reserved[0] is not a reserved byte if VAConfigAttribValDecJPEG is union. So I think you should define VAConfigAttribValDecJPEG as struct instead of union. 

Please add the comment that the reserved byte must be 0, otherwise we will have issue when re-using these bytes in the future.